docs/docs/specs/v0.2/graph.md

```diff
@@ -0,0 +1,122 @@
+# Graph API
+a proposal for `plugin.provides.graph`
+
+```tsx
+// let's say nodes are like this
+export type Node<T = any> = {
+  id: string;
+  label: string;
+  data: T;
```

Comment:
Do we anticipate it could cause perf issues if we resolve _all the data_ the UI could possibly display on init?